### PR TITLE
Add Tryton to project list

### PR DESCRIPTION
**Pull request by cedk**
The Tryton project is already Pyhton3 compatible but we plan to drop Python2 in the [next release](https://bugs.tryton.org/issue3211).

### CODE REVIEW

File: _sections/30-projects.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/osbrain.png)osBrain](https://github.com/opensistemas-hub/osbrain)
 - [![](assets/pymeasure.png)PyMeasure](https://github.com/ralph-group/pymeasure)
 - [![](assets/chaquopy.png)Chaquopy](https://chaquo.com/chaquopy/)
+- [![](assets/tryton.png)Tryton](http://www.tryton.org/)
```
Comment (takluyver) [inline _sections/30-projects.md]: I think Tryton could probably be somewhere higher up the list - the idea is that projects are ordered by prominence. We haven't really got any good way to decide exactly where to put a project that's not on Github, though.

**Comment by takluyver** [comment]
Thanks! Just to check because I'm not familiar with Tryton's development process: are you a core developer, and are you satisfied that there's a consensus in the development team to sign on to this statement?

**Comment by cedk** [comment]
Yes, I'm BDFL for Tryton project. And we already stated in our last release news that it was the last one supporting Python2: http://www.tryton.org/posts/new-tryton-release-48.html

**Comment by takluyver** [comment]
Thanks, and welcome aboard! :-)

**Comment by Carreau** [comment]
Great ! DOn't forget to look at the [practicalities](http://python3statement.org/practicalities/) section, set the `python_requires` metadata and fail with a clean error message on Python 2.

**Comment by cedk** [comment]
I will, thanks.

**Pull request merged**